srcs/app/server/WebServer.cpp

```diff
@@ -14,16 +13,18 @@ WebServer::~WebServer() {
 }
 
 void	WebServer::Run() {
-	AddListeningSocketsToMasterSet_();
-	while (max_sd_ != -1) {
+	while (fdSets.getMaxSocket() != -1) {
 		int ready_sockets =
-			select(max_sd_ + 1, fdSets.getReadSet(), fdSets.getWriteSet(), NULL, NULL);
+			select(
+				fdSets.getMaxSocket() + 1,
```

Comment:
We could encapsulate that + 1 inside the fdSets.getMaxSocket, couldn't we? Are there instances where we use the raw max socket without the + 1, (besides the one explained below) ?

Reply:
For example, three lines above, we use it without adding anything...